tests/integration_tests/test_ray.py

```diff
@@ -782,3 +784,104 @@ def test_ray_preprocessing_placement_group(tmpdir, ray_cluster_2cpu):
             skip_save_log=True,
         )
         preds, _ = model.predict(dataset=dataset)
+
+
+@pytest.mark.distributed
```

Comment:
Super thorough tests -- thanks for adding them in!

Reply:
Thanks!